src/main/java/com/minecolonies/network/messages/OpenInventoryMessage.java

```diff
@@ -96,15 +156,27 @@ public IMessage onMessage(OpenInventoryMessage message, MessageContext ctx)
             case INVENTORY_CITIZEN:
                 InventoryCitizen citizenInventory = ((EntityCitizen) player.worldObj.getEntityByID(message.entityID)).getInventoryCitizen();
                 if(!StringUtils.isNullOrEmpty(message.name))
-                    citizenInventory.setCustomName(message.name);   //SetInventoryName
+                {
+                    citizenInventory.setCustomName(message.name);
+                }
                 player.displayGUIChest(citizenInventory);
                 break;
             case INVENTORY_CHEST:
                 TileEntityChest chest = (TileEntityChest) BlockPosUtil.getTileEntity(player.worldObj, message.tePos);
                 if(!StringUtils.isNullOrEmpty(message.name))
-                    chest.setCustomName(message.name);              //SetInventoryName
+                {
+                    chest.setCustomName(message.name);
+                }
                 player.displayGUIChest(chest);
                 break;
+            case INVENTORY_FIELD:
+                InventoryField inventoryField = ColonyManager.getColony(colonyId).getField(message.tePos).getInventoryField();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'inventoryField' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)